pull_data.R

```diff
@@ -0,0 +1,33 @@
+library(sbtools)
+library(data.table)
+library(readxl)
+
+source("auth.R")
```

Comment:
I'm unfamiliar with sbtools, but is this a file that is not checked in that has the sciencebase credentials?